platforms/pf-pami/motion_control.cpp

```diff
@@ -126,7 +126,8 @@ static cogip::motion_control::PosePIDController linear_pose_controller(&linear_p
 static cogip::motion_control::SpeedFilterParameters linear_speed_filter_parameters(
     platform_min_speed_linear_mm_per_period,
     platform_max_speed_linear_mm_per_period,
-    platform_max_acc_linear_mm_per_period2
+    platform_max_acc_linear_mm_per_period2,
+    false
```

Comment:
It could be set defined as a constant with an explicit name.

Reply:
Same here, I deliberately did not port the modification as this platform will move to trash and the `pf-robot-motion-control` will be used as reference